test/input_gen/genInput.py

```diff
@@ -51,6 +51,12 @@ def save(filename, *data):
 def gen_tensor(shape, dtype=dtypes.float32):
   return np.random.random_sample(shape)
 
+##
+# @brief generate random tensor
+def gen_tensor_int(shape):
+  rand_t = np.random.randint(1, 100, size=shape)
```

Comment:
lets make it (1, 10)
1, 100 causes overflow sometimes

Reply:
This change is related with another PR #494